gateway/mw_jwt_test.go

```diff
@@ -382,6 +382,25 @@ func TestJWTSessionRSABearer(t *testing.T) {
 	})
 }
 
+
+func TestJWTSessionFailRSA_WrongJWT_Signature(t *testing.T) {
+	ts := StartTest(nil)
+	defer ts.Close()
+
+	//default values, same as before (keeps backward compatibility)
+	ts.prepareGenericJWTSession(t.Name(), RSASign, KID, false)
+	authHeaders := map[string]string{"authorization": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"}
+
+	t.Run("Request with invalid JWT signature", func(t *testing.T) {
+		ts.Run(t, test.TestCase{
```

Comment:
linter complains here, maybe we should do this: `_, _ = ts.Run....`

Reply:
will check

Reply:
it's weird since it's not complaining on other parts with same code